pkg/tsdb/influxdb/influxql/converter/testdata/influx_select_value_from_*-frame.jsonc

```diff
@@ -0,0 +1,128 @@
+//  🌟 This was machine generated.  Do not edit. 🌟
+//  
+//  Frame[0] 
+//  Name: cpu.mean
+//  Dimensions: 2 Fields by 4 Rows
+//  +-------------------------------+-------------------+
+//  | Name: Time                    | Name: Value       |
+//  | Labels:                       | Labels:           |
+//  | Type: []time.Time             | Type: []float64   |
+//  +-------------------------------+-------------------+
+//  | 2023-10-22 14:25:00 +0000 UTC | 986.6810475626996 |
+//  | 2023-10-22 14:30:00 +0000 UTC | 988.37928300062   |
+//  | 2023-10-22 14:35:00 +0000 UTC | 994.4688190880804 |
+//  | 2023-10-22 14:40:00 +0000 UTC | 988.5477466196328 |
+//  +-------------------------------+-------------------+
+//  
+//  
+//  
+//  Frame[1] 
+//  Name: derivative.mean
+//  Dimensions: 0 Fields by 0 Rows
+//  +
```

Comment:
this looks like it is dropping the additional results

Reply:
I don't get that. Which one is it dropping?

Reply:
I fixed this in latest commits.